heat_and_trees/heat_and_trees.ipynb

```diff
@@ -446,7 +427,7 @@
    "source": [
     "## Calculate land surface temperature\n",
     "\n",
-    "Given the NDVI calculated above, we can determine land surface temperature. For ease, we'll use some *top of atmosphere* calculations that have already been written up as Python functions as part of rasterio work in the `rio_toa` module. We'll also need to specify one more for transforming satellite temperature (brightness temp) to land surface temperature. For these calculations we'll use both Thermal Infrared bands - 10 and 11."
+    "Given the NDVI calculated above, we can determine land surface temperature. For ease, we'll use some *top of atmosphere* calculations that have already been written up as Python functions as part of rasterio work in the [rio_toa](https://github.com/mapbox/rio-toa/tree/master) module. We will reproduce the `brightness_temp` and `toa_utils` functions from the `rio_toa` module here to make it easier and faster."
```

Comment:
I don't it's true we make it *faster*? We only avoid one import but that doesn't make a difference really.

Reply:
IIRC, the reason for avoiding the imports is because the `rio-toa` module has not been maintained for long (about 8 years), so we wanted to avoid any potential breaking issues with the module.

Reply:
Yes. And this package doesn't exist on the defaults channel so we had to pull it from conda-forge. There's nothing wrong with conda-forge, it's just that for these examples we prefer to have them depend on the defaults channel whenever possible.

Reply:
I would just remove the *faster* reference in your update, I don't think we're making anything significantly faster here.